FIFTYONE_TEAMS.md

```diff
@@ -0,0 +1,62 @@
+# Deploying VoxelGPT to FiftyOne Teams
+
+This guide explains how to upload the latest version of VoxelGPT to your
+[FiftyOne Teams](https://voxel51.com/fiftyone-teams) deployment.
+
+## Uploading the code
+
+You can upload VoxelGPT manually or via the
+[Management SDK](https://docs.voxel51.com/teams/management_sdk.html).
+
+### Manual upload
+
+First clone and ZIP the repository:
+
+```shell
+git clone https://github.com/voxel51/voxelgpt
```

Comment:
There's a link for this. Most other repos just link to that instead of having the user do it manually.

Reply:
https://github.com/voxel51/voxelgpt/archive/refs/heads/main.zip

Reply:
Sweet 👍